### DISCUSSION TITLE
Allow players to hide the score change animation at the end of the song

### DISCUSSION THREAD

**Discussion by guanjiao23456** [ppy/osu#27373] [Ideas] [answered]
After a recent update, when a song is over players can see their PP and ranking change in the top right corner, I suggest that this animation can be hidden in the settings
I for myself don't like this animation because I don't care much about PP, I think the game itself is more important than the scores and rankings, and when I see my rankings change I'm either sad because it's lowered or I want better scores even more when it's raised.
Although there should be very few people who don't care much about PP, I'd still like to add this feature

Machine translation was used and may not be accurate, so please understand

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/27287
